src/WaCore.Crud.Web/ViewModels/WacListDataVm.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using WaCore.Crud.Contracts.Web;
+using WaCore.Crud.Dtos.Filters;
+
+namespace WaCore.Crud.Web.ViewModels
+{
+    public class WacListDataVm<TDto, TFilter> : IWacListDataVm<TDto, TFilter>
+        where TDto : class
+        where TFilter : WacFilter
+    {
+        public List<TDto> DtoList { get; set; }
```

Comment:
Here as well, I suggest to use IList or something more common.

Reply:
and again  same as above...